traits/observation/api.py

```diff
@@ -8,12 +8,14 @@
 #
 # Thanks for using Enthought open source!
 
-from traits.observation.exception_handling import (     # noqa: F401
```

Comment:
The '# noqa's here are unnecessary - they're taken care of by the `per-file-ignores` setting in `setup.cfg`.

Reply:
I'll revert here and make these changes in a separate PR.